docs/4.0/docs/design/private-registry.md

```diff
@@ -1,12 +1,13 @@
 # How to use sealos private registry
 
-All cluster nodes have been configure priave registry, When pulling an image, it will pull from the private image registry,You can use the follow command view the configuration on any cluster node:
+## How it works
+
+Each node will run an image-cri-shim daemon,The Kubelet performs grpc interaction with image-cri-shim when pulling the image.It finds the image in the private registry according to the image name.If the image exist it pulls from local,otherwise pull from remote.
```

Comment:
```suggestion
Each node will run an `image-cri-shim` daemon, the `Kubelet` performs `grpc` interaction with `image-cri-shim` when pulling the image. Witch will find the image in the private registry according to the image name. If the image exists, it will pulls from local, otherwise pull from remote.
```